OsmAnd/res/values/strings.xml

```diff
@@ -1691,14 +1691,14 @@
     <string name="recording_context_menu_arecord">Take an audio note</string>
     <string name="recording_context_menu_vrecord">Take a video note</string>
     <string name="layer_recordings">Recording layer</string>
-    <string name="recording_can_not_be_played">Recording can not be played</string>
+    <string name="recording_can_not_be_played">Could not play recording.</string>
     <string name="recording_context_menu_delete">Delete recording</string>
     <string name="recording_context_menu_play">Play</string>
     <string name="recording_description">Recording %1$s %3$s %2$s</string>
     <string name="recording_default_name">Recording</string>
     <string name="map_widget_av_notes">Audio/video notes</string>
     <string name="map_widget_distancemeasurement">Distance measurement</string>
-    <string name="audionotes_location_not_defined">Location to associate with the note is not defined yet. \"Use location…\" to assign a note to the location specified.</string>
+    <string name="audionotes_location_not_defined">Tap \'Use location…\' to add a location to the note.</string>
```

Comment:
It is not clear whether a note is added to a location or vice versa. Original does both.